rust/connlib/tunnel/src/peer.rs

```diff
@@ -699,6 +743,10 @@ fn mapped_ipv6(ips: &[IpAddr]) -> Vec<IpAddr> {
     }
 }
 
+fn is_dns_addr(addr: IpAddr) -> bool {
+    IpNetwork::from(IPV4_RESOURCES).contains(addr) || IpNetwork::from(IPV6_RESOURCES).contains(addr)
+}
+
 #[cfg(test)]
 mod tests {
```

Comment:
Would it be very difficult to at least add a unit-test? I think some kind of automated test would be good, doesn't have to be a proptest.

Reply:
I can do it, I'd need to replicate the whole proxy-ips thingy to have it mean something. Let me take another look at this.

Reply:
> I can do it, I'd need to replicate the whole proxy-ips thingy to have it mean something. Let me take another look at this.

It shouldn't be that difficult, should it? All we need is to define a few IP addresses, doesn't have to be 8, 1 proxy IP should be enough, right?